newrelic-agent/src/test/java/com/newrelic/agent/config/HighSecurityTest.java

```diff
@@ -162,7 +126,6 @@ public void isEnableHighSecuritySystemProperty() throws Exception {
             AgentConfig config = AgentConfigImpl.createAgentConfig(localMap);
 
             Assert.assertTrue(config.isHighSecurity());
-            Assert.assertTrue(config.isSSL());
             Assert.assertEquals(SqlObfuscator.OBFUSCATED_SETTING, config.getTransactionTracerConfig().getRecordSql());
```

Comment:
Would be nice while in here to pick up the static imports for asserts as a campsite.